### PR TITLE
initial metainfo cleanup and removed unused submodule

### PR DESCRIPTION
**Pull request by xanscale**
- metainfo cleanup applying linter suggestion
- removed shared-module git submodule due to be unused from Chromium.BaseApp

@refi64

### CODE REVIEW

File: com.google.Chrome.metainfo.xml
Comment (bbhtt) [inline com.google.Chrome.metainfo.xml:1]: Did you change the indentation or something in this? The diff seems wrong.

File: com.google.Chrome.metainfo.xml [hunk 1]
```diff
@@ -1,223 +1,39 @@
...
-  </description>
-  <screenshots>
-    <screenshot type="default">
-      <image>https://www.gstatic.com/chrome/appstream/chrome-2.png</image>
-    </screenshot>
-  </screenshots>
-  <releases>
-    <release version="129.0.6668.58-1" date="2024-09-16">
-      <description></description>
-    </release>
```
Comment (bbhtt) [inline com.google.Chrome.metainfo.xml:27]: These are automatically added by x-checker, not sure if removing them is worth it.

**Comment by bbhtt** [comment]
Can you also rebase instead of merging master and squash the other commits in to two ?

**Comment by xanscale** [comment]
@refi64 @bbhtt

**Pull request merged**